.github/workflows/ios_build.yml

```diff
@@ -53,6 +53,11 @@ jobs:
           name: golden-failures
           path: app/test/**/failures/*
 
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
```

Comment:
Just a question: Do you think it maybe makes sense to pin the xcode version her?